tests-e2e/cypress/integration/frontstage/incident_dialog_spec.js

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+describe('incident creation dialog', () => {
+    const playbookName = 'Playbook (' + Date.now() + ')';
+
+    before(() => {
+        // # Login as user-1
+        cy.apiLogin('user-1');
+
+        // # Create a playbook
+        cy.apiGetTeamByName('ad-1').then((team) => {
+            cy.apiGetCurrentUser().then((user) => {
+                cy.apiCreateTestPlaybook({
+                    teamId: team.id,
+                    title: playbookName,
+                    userId: user.id,
+                });
+            });
+        });
+    });
+
+    beforeEach(() => {
+        // # Login as user-1
+        cy.apiLogin('user-1');
+
+        // # Navigate to the application
+        cy.visit('/');
+
+        // # Trigger the incident creation dialog
+        cy.openIncidentDialogFromSlashCommand();
+
+        // * Verify the incident creation dialog has opened
+        cy.get('#interactiveDialogModal').should('be.visible').within(() => {
+            cy.findByText('Incident Details').should('be.visible');
+        });
+    });
+
+    it('cannot create without filling required fields', () => {
```

Comment:
Maybe something like this?
```suggestion
    it('cannot create an incident without filling required fields', () => {
```
This way it reads `incident creation dialog cannot create an incident without filling required fields`.